scenario-examples-bootstrapper/utils/nu/deploy-scenario-and-wait-for-deployed-state.sh

```diff
@@ -143,9 +145,15 @@ if [[ "$DEPLOYMENT_STATUS" == "RUNNING" ]]; then
 elif [[ "$DEPLOYMENT_STATUS" == "CANCELED" || "$DEPLOYMENT_STATUS" == "NOT_DEPLOYED" ]]; then
   deploy_scenario "$SCENARIO_NAME"
 elif [[ "$DEPLOYMENT_STATUS" == "DURING_DEPLOY" || "$DEPLOYMENT_STATUS" == "FINISHED" ]]; then
-  # Do nothing, skip to waiting for RUNNING or FINISHED state
+  # Do nothing, skip to waiting for RUNNING or FINISHED status
   # TODO: For DURING_DEPLOY: Cancel and then deploy when the deployed version is different than expected. As of now, NEW_SCENARIO_VERSION may be null here because Nussknacker doesn't return this information for DURING_DEPLOY status.
   :
+elif [[ "$DEPLOYMENT_STATUS" == "PROBLEM" ]]; then
```

Comment:
I thought about adding `&& "$STATUS_DESCRIPTION" != "Failed to get a state of the scenario."`